test/test_cluster_monitor.py

```diff
@@ -5,12 +5,19 @@
 See LICENSE for details
 """
 
+from contextlib import closing
+from datetime import datetime, timedelta
+from distutils.version import LooseVersion
```

Comment:
Cf. https://github.com/aiven/pglookout/pull/82 It's probably better to not reintroduce deprecated API usage.